tests/unit_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -159,6 +196,32 @@ def test_get_deactivated_participants(self,
 
         self.assertEqual(expected_response, dataset_response)
 
+    @mock.patch('utils.participant_summary_requests.get_participant_data')
+    def test_get_site_participant_information(self, mock_get_participant_data):
+
+        # Pre conditions
+        mock_get_participant_data.return_value = self.site_participant_info_data
+        site_participant_information = []
+
+        for entry in mock_get_participant_data.return_value:
+            item = []
+            for col in psr.FIELDS_OF_INTEREST_FOR_VALIDATION:
+                for key, val in entry.get('resource', {}).items():
+                    if col == key:
+                        item.append(val)
+            site_participant_information.append(item)
+
+        # Tests
+        expected_dataframe = pandas.DataFrame(
+            site_participant_information,
+            columns=psr.FIELDS_OF_INTEREST_FOR_VALIDATION)
+        actual_dataframe = pandas.DataFrame(
+            self.site_participant_information,
+            columns=psr.FIELDS_OF_INTEREST_FOR_VALIDATION)
+
+        # Post conditions
+        pandas.testing.assert_frame_equal(expected_dataframe, actual_dataframe)
```

Comment:
This unit test is not testing your code right now.  It never executes the `get_site_participant_information` function.  It is comparing identical versions of the dataframe, so it always passes.  I've attached a patch file that runs the function and adds some print statements for debugging purposes.  

[unit_test_patch.txt](https://github.com/all-of-us/curation/files/5712553/unit_test_patch.txt)